packages/schematics/angular/class/schema.json

```diff
@@ -52,7 +52,6 @@
     }
   },
   "required": [
-    "name",
-    "project"
```

Comment:
Shouldn't project here still be required? It will error out inside `createDefaultPath` if it's not there.

Reply:
The smart default will provide a value if not specified so it’s not actually required to be provided by the user.  The schema type generator doesn’t know about smart defaults though so it gives the property an optional type.  Marking it as required “fixes” the schema type generator but is technically incorrect.  This results in issues when other consumers attempt to use the schemas (UI based ng generate utilities for instance) since they will require the user to specify the property even though a value will automatically be provided.